src/main/java/net/openhft/chronicle/core/threads/EventLoop.java

```diff
@@ -84,4 +84,13 @@ static boolean inEventLoop() {
      */
     @Override
     void close();
+
+    /**
```

Comment:
maybe this should be in `net.openhft.chronicle.threads.CoreEventLoop`?

Reply:
We want this to have as high in interface hierarchy as possible to minimize amount of casting. EventLoop exists in 99% cases to be a proxy for CoreEventLoop anyway.